prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -1,3 +1,42 @@
+presubmits:
+  GoogleCloudPlatform/osconfig:
+    - name: presubmit-gocheck
```

Comment:
Please at least include `osconfig` in the presubmit name to avoid naming collisions if we add a similar job to another repo. (Same with `presubmit-gobuild` below.)